fuzzers/005-tilegrid/generate_full.py

```diff
@@ -221,6 +221,99 @@ def propagate_INT_bits_in_column(database, tiles_by_grid):
             tile = database[tile_name]
 
 
+def propagate_INT_INTERFACE_bits_in_column(
+        database, tiles_by_grid, int_interface_name):
+    """ Propigate INT offsets up and down INT columns.
```

Comment:
Is this comment correct?  Feels like a copy paste, but unclear?

Reply:
Yeah, this function is similar to the `propagate_INT_bits_in_column`. I have updated the comment which was a leftover, but I might work some more to try and de-duplicate the code, if possible.